cpp/src/parquet/file_serialize_test.cc

```diff
@@ -346,12 +346,12 @@ TEST(TestBufferedRowGroupWriter, DisabledDictionary) {
 }
 
 TEST(TestBufferedRowGroupWriter, MultiPageDisabledDictionary) {
-  const int VALUE_COUNT = 10000;
-  const int PAGE_SIZE = 16384;
+  const int value_count = 10000;
```

Comment:
small nit as long as you are changing this, these should probably be constexpr kValueCount, etc.

Reply:
And moved to constexpr while were at it.